Makefile

```diff
@@ -0,0 +1,43 @@
+.SUFFIXES:
+
+CC := hugo --minify --gc
+TEST := hugo server --baseURL http://localhost/ --port 1313
```

Comment:
blog must be /blog/ or some paths will not work. For example `/blog/favicon.png`.

Reply:
fixed at 0fd8e4ea